vimeo/VimeoUpload
# VIM-XXXX: Updates README to specify carthage support

#### Ticket

[VIM-XXXX](https://vimean.atlassian.net/browse/VIM-XXXX)

- *Note: this is required for Vimeo staff only. If not applicable to your PR, use "N/A".*

#### Pull Request Checklist


#### Issue Summary

Updates README to specify installation methods

#### How to Test

To test Carthage support simply run `carthage build --no-skip-current`.

Once completed the `Carthage/Build` folder should exist and inside it you will find the compiled framework file under the applicable platform (iOS only in this case).
